packages/camera/camera/android/src/main/java/io/flutter/plugins/camera/features/exposurepoint/ExposurePointFeature.java

```diff
@@ -37,18 +46,13 @@ public String getDebugName() {
 
   @Override
   public Point getValue() {
-    return currentSetting;
+    return exposurePoint;
   }
 
   @Override
-  public void setValue(@NonNull Point value) {
-    this.currentSetting = value;
-
-    if (value.x == null || value.y == null) {
-      cameraRegions.resetAutoExposureMeteringRectangle();
-    } else {
-      cameraRegions.setAutoExposureMeteringRectangleFromPoint(value.x, value.y);
-    }
+  public void setValue(Point value) {
+    this.exposurePoint = value == null || value.x == null || value.y == null ? null : value;
```

Comment:
Nit: please put `()` around the condition; precedence of || vs ?: isn't something I would expect people to remember offhand. (Hopefully this wouldn't compile if it were opposite what were intended, but not being very familiar with Java I'm not 100% sure problematic type coercion couldn't happen here.)